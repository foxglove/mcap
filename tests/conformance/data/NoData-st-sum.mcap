version https://git-lfs.github.com/spec/v1
oid sha256:adfdbe9b9019beab03f7470fa54e3ec04793aff485cfe61bff3b1d7b56e75469
size 212
