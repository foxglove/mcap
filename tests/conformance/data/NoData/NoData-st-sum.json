{
  "records": [
    {"type": "Header", "fields": [["library", ""], ["profile", ""]]},
    {"type": "DataEnd", "fields": [["data_section_crc", "3959079795"]]},
    {
      "type": "Statistics",
      "fields": [
        ["attachment_count", "0"],
        ["channel_count", "0"],
        ["channel_message_counts", {}],
        ["chunk_count", "0"],
        ["message_count", "0"],
        ["message_end_time", "0"],
        ["message_start_time", "0"],
        ["metadata_count", "0"],
        ["schema_count", "0"]
      ]
    },
    {
      "type": "SummaryOffset",
      "fields": [
        ["group_length", "55"],
        ["group_opcode", "11"],
        ["group_start", "38"]
      ]
    },
    {
      "type": "Footer",
      "fields": [
        ["summary_crc", "430344505"],
        ["summary_offset_start", "93"],
        ["summary_start", "38"]
      ]
    }
  ],
  "meta": {"variant": {"features": ["st", "sum"]}}
}
