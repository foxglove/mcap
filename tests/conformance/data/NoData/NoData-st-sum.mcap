version https://git-lfs.github.com/spec/v1
oid sha256:78aed9973f50c4ebc30a8eab27453f6e42b2984c4f402d57e002dafdf217b744
size 156
