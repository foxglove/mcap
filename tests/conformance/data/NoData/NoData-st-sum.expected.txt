Header library="" profile=""
Statistics attachment_count=0 channel_count=0 channel_message_counts={} chunk_count=0 message_count=0
SummaryOffset group_length=0 group_opcode=13 group_start=71
SummaryOffset group_length=33 group_opcode=11 group_start=38
SummaryOffset group_length=0 group_opcode=10 group_start=71
SummaryOffset group_length=0 group_opcode=8 group_start=71
Footer summary_crc=0 summary_offset_start=71 summary_start=38
